### DISCUSSION TITLE
WON'T VERIFY TEACHER STATUS

### DISCUSSION THREAD

**Discussion by SAHS29803** [community/community#123309] [GitHub Education]
### Hi everyone,

"•	Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://nam12.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fsettings%2Fbilling%2Fpayment_information&data=05%7C02%7Cmliebman%40acpsd.net%7C7474e8f4a6104228feee08dc71170087%7Cf86e302d701341f499871c135ce4132b%7C0%7C0%7C638509590808637917%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C0%7C%7C%7C&sdata=32plRTM9H4xqcoWcSDdiUA0X%2FI3Z3Lg9tKHp%2FJqyDsg%3D&reserved=0). Variations of your name and nicknames are unacceptable."

My name is my name!!!!!  What's the problem?!?!?!?

**Comment by gil-son** [comment]
Hello, how are you? I feel for that situation, you need to verify it


Access the [Github Support](https://support.github.com/)

Click on My Ticket -> New Ticket -> Finally describe it

If I supported you consider to mark my answer. Good Luck!
